guidelines/REUSE.md

```diff
@@ -1,85 +1,100 @@
-```json
-{
-  "description": "A simple model",
-  "type": "object",
-  "properties": {
-    "id": {
-      "type": "integer"
-    }
-  }
-}
+When referencing locally (within the current document), the target references should follow the conventions, as defined by the spec:
+
+* Parameters -> `#/parameters`
+* Responses -> `#/responses`
+* Definitions (Models/Schema) -> `#/definitions`
+
+An example of a local definition reference:
+
+_Example from https://github.com/OAI/OpenAPI-Specification/blob/master/examples/v2.0/json/petstore.json_
+``` json
+          "200": {
+            "description": "pet response",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Pet"
+              }
+            }
 ```
 
-The reference would be `https://my.company.com/definitions/Model.json`.
+#### Remote references
 
-_Assuming file https://my.company.com/definitions/models.json_
-```json
-{
-  "models": {
-    "Model": {
-      "description": "A simple model",
-      "type": "object",
-      "properties": {
-        "id": {
-          "type": "integer"
-        }
-    },
-    "Tag": {
-      "description": "A tag entity in the system",
-      "type": "object",
-      "properties": {
-        "name": {
-          "type": "string"
-        }
-      }
-    }
-  }
+##### Relative path
+
+Files can be referred to in relative paths to the current document. 
+
+_Example from https://github.com/OAI/OpenAPI-Specification/tree/master/examples/v2.0/json/petstore-separate/spec/swagger.json_
+
+``` json
+"responses": {
+	"default": {
+		"description": "unexpected error",
+		"schema": {
+			"$ref": "../common/Error.json"
+		}
+	}
 }
 ```
 
-The reference to the `Model` model would be `https://my.company.com/definitions/models.json#/models/Model`. Make sure you include the full path to the model itself, including its containers if needed.
+Remote references may also reference properties within the relative remote file.
+
+_Example from https://github.com/OAI/OpenAPI-Specification/tree/master/examples/v2.0/json/petstore-separate/spec/swagger.json_
+``` json
+"parameters": [
+	{
+		"$ref": "parameters.json#/tagsParam"
+	},
+	{
+		"$ref": "parameters.json#/limitsParam"
+	}
+]
+```
+
 
-Whether you reference definitions internally or externally, you can never override or change their definitions from the referring location. The definitions can only be used as-is.
+##### URL
 
-### Definitions
-Reuse schema definitions by creating a repository of definitions.  This is done by simply hosting a file or set of files for commonly used definitions across a company or organization.  In the case of multiple files, the models can be referenced directly as such:
+Remote files can be hosted on an HTTP server (rather than the local file system). 
+
+One risk of this approach is that environment specific issues could arise if DNS is not taken into account (as the reference can only contain one hostname).
```

Comment:
But why would someone expect it to be if there is no mention of this anywhere except for path parameters?  I'm not against keeping it, I just thought by adding the details that it could be confusing.  Your call.